core/src/main/java/org/apache/calcite/plan/volcano/VolcanoPlanner.java

```diff
@@ -64,19 +45,8 @@
 
 import java.io.PrintWriter;
 import java.io.StringWriter;
-import java.util.ArrayDeque;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Comparator;
-import java.util.Deque;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.IdentityHashMap;
-import java.util.Iterator;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.text.NumberFormat;
+import java.util.*;
```

Comment:
Please expand this to avoid style problems?

Reply:
The style check passed without problem.

Reply:
I think it's not expected considering the calcite code convention.

Reply:
Avoid to import star which is a bad code practice.

Reply:
Fixed. We need to fix the style check as well. Such things should be enforced by tool.